quickfixj-core/src/test/java/quickfix/SessionTest.java

```diff
@@ -3145,4 +3145,47 @@ public void testSend_ShouldKeepPossDupFlagAndOrigSendingTime_GivenAllowPosDupCon
         assertTrue(sentMessage.getHeader().isSetField(PossDupFlag.FIELD));
         assertTrue(sentMessage.getHeader().isSetField(OrigSendingTime.FIELD));
     }
+
+
+    @Test
```

Comment:
What does this test do if you roll back your changes in the Session class?

Reply:
I don't know what you mean